airflow/utils/cli.py

```diff
@@ -46,14 +46,10 @@
 
 def _check_cli_args(args):
     if not args:
-        raise ValueError("Args should be set")
-    if not isinstance(args[0], Namespace):
-        raise ValueError(
-            f"1st positional argument should be argparse.Namespace instance, but is {type(args[0])}"
-        )
+        raise ValueError(f"Args should be set: {args} [{type(args)}]")
 
 
-def action_cli(func=None, check_db=True):
+def action_cli(func=None, check_db=True, check_cli_args=True):
```

Comment:
The new `check_cli_args` flag is used [here](https://github.com/apache/airflow/pull/22613/files#diff-2bddc31b61cb7aa1de96c8e41d506d6c9d76345db432893ce55d3ec75e87e2fcR96).